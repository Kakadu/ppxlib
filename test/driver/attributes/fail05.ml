let x = (42 [@foo])
