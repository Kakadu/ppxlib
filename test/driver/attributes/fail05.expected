File "fail05.ml", line 1, characters 14-17:
Error: Attribute `foo' was silently dropped
